data/Contoso_Electronics_Company_Overview.md

```diff
@@ -0,0 +1,48 @@
+# Contoso and Sons
```

Comment:
Can you match the company name/type used in the PDFs?

Reply:
Done